server/src/main/resources/authorizationConfig.yml

```diff
@@ -4,13 +4,190 @@ metaStoreOptions:
   configFile: "src/main/resources/zkConfig.yml"
 
 roleDefinitions:
-  project.admin:
-    roleId: project.admin
+  org.admin:
+    roleId: org.admin
+    permissions:
+      - ORG_CREATE
+      - ORG_UPDATE
+      - ORG_DELETE
+      - ORG_GET
+      - ORG_LIST
+      - ORG_PROJECT_MIGRATE
+      - TEAM_CREATE
+      - TEAM_UPDATE
+      - TEAM_DELETE
+      - TEAM_GET
+      - TEAM_LIST
+      - PROJECT_CREATE
+      - PROJECT_UPDATE
+      - PROJECT_DELETE
+      - PROJECT_LIST
+      - PROJECT_GET
+      - TOPIC_CREATE
+      - TOPIC_UPDATE
+      - TOPIC_DELETE
+      - TOPIC_GET
```

Comment:
TOPIC_LIST ?
Not sure if possible but it would be good to have these defined as additive (i believe yaml doesn't allow merge of list anchors). Any other way to define it ?

Reply:
Tried using merge keys in yaml but it seems jackson does not recognize the `<<:` anchor